biz.aQute.tester.junit-platform/src/aQute/tester/junit/platform/Activator.java

```diff
@@ -348,11 +344,21 @@ LauncherDiscoveryRequest buildRequest(List<? extends DiscoverySelector> selector
 	long test(LauncherDiscoveryRequest testRequest) {
 		trace("testing request %s", testRequest);
 		try {
+			ServiceTracker<TestExecutionListener, TestExecutionListener> track = new ServiceTracker<>(context,
+				TestExecutionListener.class, null);
+			track.open(true);
```

Comment:
You do not want `true` here. Using `true` means you want to track all services including services which are not type compatible with your bundle. So you could track a `TestExecutionListener` service which this bundle cannot cast to `TestExecutionListener`.

Just use `track.open();`

Reply:
Done, thanks (I obviously misunderstood the docs around `open(true)`.)